azurelinuxagent/ga/update.py

```diff
@@ -270,7 +293,7 @@ def _ensure_latest_agent(self):
         if len(manifests) == 0:
             msg = u"Incarnation {0} has no agent family {1} updates".format(etag, family)
             logger.info(msg)
-            add_event(u"WALA", message=msg)
+            add_event(AGENT_NAME, version=CURRENT_VERSION, message=msg)
```

Comment:
I was not suggesting an op=xxxx, I was more suggesting a method that does *not* take an op to better convey that no op is not associated with the event.  I'm trying to distinguish between logging messages, and logging events.  This code does the latter, and I cannot think of a good way to meaningfully distinguish.

Spitballing... 
Maybe call it log_(info|warn|error)_event(...)

It seems like 286 and 296 would be very chatty especially considering they are events.  How frequently is this code evaluated?  Is there any value in having them as events?  Is there value is creating an event for INFO messages, my gut says probably not otherwise they wouldn't be INFO.